samples/JSON with Comments/person.jsonc

```diff
@@ -0,0 +1,24 @@
+// example comment explaining schema
```

Comment:
Where is this sample from? It seems a bit too simple...

Reply:
It's a copy of the [person.json](https://github.com/github/linguist/blob/master/samples/JSON/person.json) from the plain `.json` samples with the added comment. I figured trying to add more comments might be contrived.

[Here's the file](https://github.com/nathanshelly/.files/blob/master/neovim/coc.jsonc) that prompted me opening this PR. I could add it as the sample if it's better or happy to do whatever suggestion you might have have!

Reply:
> [Here's the file](https://github.com/nathanshelly/.files/blob/master/neovim/coc.jsonc) that prompted me opening this PR. I could add it as the sample if it's better or happy to do whatever suggestion you might have have!

Yes, please do. That one is much better :-)

Reply:
updated with the better sample 😄